src/test/java/com/wikia/webdriver/elements/communities/desktop/pages/SpamWikiReviewPage.java

```diff
@@ -0,0 +1,68 @@
+package com.wikia.webdriver.elements.communities.desktop.pages;
+
+import com.wikia.webdriver.pageobjectsfactory.pageobject.BasePageObject;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+import java.util.Collections;
+import java.util.List;
+
+public class SpamWikiReviewPage extends BasePageObject {
+
+  private static final String SPAM_WIKI_REVIEW_SERVICE_NAME = "spam-wiki-review";
+  private static final String VIEW_WIKIS_SUBPAGE = "/wiki";
+
+  public static enum LANGUAGE_CODE {
+    all, en, ar, de, es, fi, fr, id, hi, hu, it, ja, ka, kn, ko, nl, pt, pl, ru, uk, vi, zh, other
+  }
+
+  @FindBy(xpath = "//button[contains(text(),'Select language of Wikis')]")
+  private WebElement selectLanguageButton;
+
+  @FindBy(xpath = "//table[@class='table table-hover']//tbody")
+  private WebElement displayedWikisTable;
+
+  public SpamWikiReviewPage open() {
+    getUrl(urlBuilder.getUrlForService(SPAM_WIKI_REVIEW_SERVICE_NAME)+VIEW_WIKIS_SUBPAGE);
+    return this;
+  }
+
+  /**
+   * This selects filtering of displayed wikis with selectLanguagesOfWikis button
+   * and loads a new page with the filter applied
+   * @param language one of LANGUAGE_CODE enum
+   */
+  public SpamWikiReviewPage selectLanguageOfWikis(LANGUAGE_CODE language){
+    selectLanguageButton.click();
+    selectLanguageButton
+        .findElement(By.xpath(String.format("//a[contains(text(),'%s')]", language)))
+        .click();
+
+    return this;
+  }
+
+  /**
+   * This gets a list of WebElements containing Rows of displayed table with Wikis' info
+   * excludes thead, header of the table
+   */
+  public List<WebElement> getListDisplayedWikisTableRows() {
+    try {
+      return displayedWikisTable.findElements(By.xpath("./tr"));
+    }
+    // if no elements were found return an empty list
+  catch (org.openqa.selenium.NoSuchElementException notFoundException) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Move this "catch" on the same line that the previous closing curly brace. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3ARightCurlyBraceSameLineAsNextBlockCheck)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Either log or rethrow this exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1166)